docs/tutorial_bcc_python_developer.md

```diff
@@ -598,30 +598,30 @@ This program instruments a user-defined static tracing (USDT) probe, which is th
 
 ```
 # ./nodejs_http_server.py 24728
-TIME(s)            COMM             PID    ARGS
-24653324.561322998 node             24728  path:/index.html
-24653335.343401998 node             24728  path:/images/welcome.png
-24653340.510164998 node             24728  path:/images/favicon.png
+TIME(s)			COMM			 PID	ARGS
```

Comment:
This is what the tool actually outputs, we should change this, right?

Reply:
Whoops, I didin't mean to edit any tool outputs.